src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -210,6 +210,15 @@ Assembly TryLoadAssembly(AssemblyName assemblyName)
                         return assembly;
                     }
 
+                    // Finds and loads NuGet assemblies if msbuildPath is in a VS installation
+                    targetAssembly = Path.GetFullPath(Path.Combine(msbuildPath, "..", "..", "..", "Common7", "IDE", "CommonExtensions", "Microsoft", "NuGet", assemblyName.Name + ".dll"));
```

Comment:
Instead of hardcoding this relative path, should we add an argument 

```C#
public static void RegisterMSBuildPath(string msbuildPath, string[] additionalDirectories)
```

And use that in `RegisterInstance`?